### PR TITLE
Fix requirements to match our usage

### PR DESCRIPTION
**Pull request by jcpunk**
Steve noted a missing requirement on `requests` and set the version of `urllib3` we need to account for bug fixes.

Working on #349

### CODE REVIEW

**Review by StevenCTimm — COMMENTED** [review]
This matches the requirements I have on the spreadsheet.
Would suggest requiring > 9.0.0 for htcondor nowadays..8.9.11 was a dev branch 
now superseded by a production release

**Comment by jcpunk** [comment]
Pushed update with higher condor version.

File: setup.py [hunk 1]
```diff
@@ -25,10 +25,11 @@
...
     "google-api-python-client >= 1.12.8",
     "gcs-oauth2-boto-plugin >= 2.7",
-    "htcondor >= 8.9.11",
-    "urllib3 >= 1.25.6",
-    "bill-calculator-hep >= 0.1.2",
+    "htcondor >= 9.0.0",
     "structlog >= 21.1.0",
+    "requests >= 2.14.2",
+    "urllib3 >= 1.26.2",
+    "bill-calculator-hep >= 0.1.2",
```
Comment (mambelli) [inline setup.py:32]: bill-calculator-help should be 0.1.4, Maria made some fixes to it (renamed the package). Bruno updated the imports but it seems we missed this.

**Review by mambelli — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (jcpunk) [reply setup.py]: Pushed update.

**Review by mambelli — APPROVED** [review]

**Pull request merged**